### ISSUE TITLE
Issues with carets on comments

### ISSUE THREAD

**Issue by LingMan** [StylishThemes/GitHub-Dark#1233]
* **Browser**: Firefox 81
* **Operating System**: Manjaro
* **GH-Dark version**: 4.2.2
* **Screenshot**:
![Screenshot from 2020-10-18 09-45-39](https://user-images.githubusercontent.com/18645382/96361559-07f7a900-1127-11eb-9d5e-c538e118f798.png)

There are three closely related issues with the carets pointing from a comment to the user's image. I hope it's fine to combine them into one report.

**### 1)**
* **Page with the issue**: Any issue or pull request with a post from you.

The caret is filled with the same color as the comment's outline. On vanilla GitHub the color matches the filling color of the comment's header instead.

**### 2)**
* **Page with the issue**: Any issue or pull request with a post from someone else.

The caret is missing entirely.

**### 3)**
* **Page with the issue**: Any link pointing directly to a comment, e.g. https://github.com/StylishThemes/GitHub-Dark/pull/1213#issue-474080561.

Basically the same as the first problem. Now that there's the glowy outline the caret appears even though the comment is from someone else, but it's still filled with the wrong color.

**Comment by LingMan** [comment]
Thanks for the quick fix!
